Frontpage text

finnshort opened the pull request:
This PR is a collection of small front page styling changes, mostly things that I thought would increase readability
- increase font size for map about legend and popups
- increase size of close button on popup
- increase size of about legend and add scrolling 
- rename 'filter' to 'date filter' to be more clear
- move language change buttons to dropdown menu: harder to find ('language' in english) but clears up a lot of space in the crowded navbar.. thoughts? alternative could be moving social media icons to bottom of map legend or map

dgboss commented:
I see this PR is still in a 'draft' state. Would you like me to take a look at it as is? Or do you have more work planned?

finnshort commented:
@dgboss missed your comment- it was not ready but is now!

I prematurely merged the content-changes branch into this one because I knew there would be conflicts and I wanted this to be ready to go once you've reviewed. But please consider it blocked by #65 .

dgboss commented:
This is looking great! I really like the changes and just have a couple suggestions/observations:
1. What do you think about the WalkRollMap.org site opening in a new tab when clicked?
2. Should we have a tiny bit of whitespace between the Strava ridership legend swatch and the words 'less' and 'more'? I think a px or two might be nice.
3. I noticed that when I open Chrome DevTools and simulate a mobile device and reload the page I can't open the legend panel. Not sure if it is an artifact of the Chrome DevTools or something else?

finnshort commented:
@dgboss sweet! I changed the walkrollmap button to open in a new tab and added some space around the strava gradient bar. The legend panel opens and closes for me in mobile device mode in devtools- clicking on the layers button opens it and clicking anywhere in the panel closes it. Do you have any other details that could help me reproduce your issue, maybe the specific mobile device type?

dgboss commented:
Looks like I'm only having the issue with being unable to open the legend when using the iPhone 12 Pro device. Other devices work great! Sorry for the false alarm. I imagine it's probably an bug with the iPhone 12 Pro simulation.

dgboss reviewed (approved):
Things look great! Thanks for fixing the hyperlink and adding the whitespace around the strava symbol in the legend!

finnshort commented:
No problem and sounds good!
